prac_05/hex_colours.py

```diff
@@ -0,0 +1,22 @@
+"""
+Lee Crawford
+CP1404/CP5632 Practical
+hex_colours
+"""
+
+HEX_COLOURS = {"aquamarine1": "#7fffd4", "black": "#000000", "BlanchedAlmond": "#ffebcd",
+               "BlueViolet": "#8a2be2", "burlywood3": "	#cdaa7d", "CadetBlue2": "#8ee5ee",
+               "chartreuse2": "#76ee00", "chocolate1": "#ff7f24", "DarkGoldenrod1": "#ffb90f",
+               "DarkOliveGreen4": "#6e8b3d"}
+
+empty_value = True
+
+colour_name = input("Enter colour name: ").lower()
+while empty_value:
```

Comment:
Shouldn't  the user keep entering  color names until they enter a blank one to stop the loop.